pkg/autodiscovery/listeners/kubelet.go

```diff
@@ -124,10 +133,59 @@ func (l *KubeletListener) processNewPods(pods []*kubelet.Pod, firstRun bool) {
 			for _, container := range pod.Status.Containers {
 				l.createService(container.ID, pod, firstRun)
 			}
+			l.createPodService(pod, firstRun)
```

Comment:
just double checking here, why do we need these new services?

Reply:
endpoints checks are resolved in the node level by node agente, so the kube pod service is required to resolve the endpoints check (adds the pod tags to the check also)

Reply:
cc @ajacquemot for the new kind of services in the AD (`pod`)